README.md

```diff
@@ -37,6 +37,14 @@ You can also [donate some
 money](https://liberapay.com/IHateMoney/donate). All funds will be used
 to maintain the [hosted version](https://ihatemoney.org).
 
+Join the Contributers already contributing to this Repository.
+
+<a href="https://github.com/spiral-project/ihatemoney/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=spiral-project/ihatemoney" />
+</a>
```

Comment:
We're using Markdown instead of HTML here, so it would be better to use it here as well :-)